test/functional/organization/cdn_configuration_test.rb

```diff
@@ -16,12 +16,14 @@
     username = "foo"
     url = "https://goo.com"
     upstream_label = "GreatOrg"
+    type = 'upstream_server'
```

Comment:
@parthaa was this change meant to go into this pr? I don't see it used anywhere

Reply:
Yeh tests were failing  without this change. We may as well merge it here.